wurst/systems/spells/mage/dementia/ActivateRunesNew.wurst

```diff
@@ -0,0 +1,117 @@
+package ActivateRunesNew
+import DarkGateNew
+import DementiaRunesNew
+import ClosureForGroups
+import Orders
+import ClosureEvents
+import DummyCaster
+import LinkedList
+import ClosureTimers
+
+@configurable constant int ABILITY_ID = 'A07B'
+@configurable constant int REQUIREMENT_ABILITY_ID = 'A07D'
+@configurable constant real PORTAL_OFFSET = 350
+@configurable constant real RUNE_CAST_RADIUS = 350
+@configurable constant string SUMMON_EFFECT_PATH = "Abilities\\Spells\\Orc\\AncestralSpirit\\AncestralSpiritCaster.mdl"
+
+LinkedList<SummonData> summonsList = new LinkedList<SummonData>()
+
+public class SummonData
+    LinkedList<RUNE_TYPE> rune_order
```

Comment:
Lets use `Rune` instead of `RUNE_TYPE` (classes and enums should be upper camel case as per the manual https://wurstlang.org/manual.html#enums)